app/utils/utils.js

```diff
@@ -37,8 +37,8 @@ function formatKeyboardShortcut (keyboardShortcut) {
 }
 
 function minutesRemaining (milliseconds) {
-  const seconds = Math.ceil(milliseconds / 1000.0)
-  const minutes = Math.ceil(seconds / 60.0)
+  const seconds = Math.round(milliseconds / 1000.0)
```

Comment:
Not sure about this change. I think I prefer the original way: I took inspiration from how Apple shows time remaining in their OS installer. 

Also, there is now different logic for Icon and Tray message, so icon can show "3" and tray "Next Mini Break in about 4 minutes"

Reply:
> Also, there is now different logic for Icon and Tray message, so icon can show "3" and tray "Next Mini Break in about 4 minutes"

My recollection was that they were not aligned before, but I checked now, and you are correct, regarding the actual behaviour. When I look at [the code for this](https://github.com/hovancik/stretchly/blob/639088e39d23a8b8155a4c4a164bc2ec552cec03/app/utils/utils.js#L15-L23C2) though, I do not understand why this is the case 🤔 (I even read the code of the library and tested locally, which led to a rounding of the value, not the ceiling I see in the app's behaviour…)

Regarding the "taste" question:
I run it with the altered logic for ~5 days now, and I really enjoy it, because I know that a `0` in the tray means I have to prepare for the break "right now", whereas the old 1 minute did not have this urgency… (I am aware of the notifications, but I tend to not notice those, and if I glance on the icon, I am probably already not fully concentrated on anything anymore).
To put it into a single sentence: The rounding gives more accurate values for the intervals' edges, which is where it counts IMHO.

Reply:
can we then make that into separate PR? 
I don't really have hard opinions on this, but I prefer it to behave same everywhere :)

Reply:
Sure, I will revert it. Should I keep in the correction of the unit test?

```diff
diff --git a/test/utils.js b/test/utils.js
--- a/test/utils.js
+++ b/test/utils.js
@@ -118,10 +118,10 @@ describe('canSkip and canPostpone', () => {

   describe('minutesRemaining', () => {
     it('one minute remaining', () => {
-      minutesRemaining(1000).should.equal(1)
+      minutesRemaining(60000).should.equal(1)
     })
     it('less then one minute remaining', () => {
       minutesRemaining(1).should.equal(0)
     })
     it('ten minutes remaining', () => {
       minutesRemaining(600000).should.equal(10)
```

Reply:
I think I was trying to test that one second will show as 1 minute, which is currently correct, right? Or there is some issue with it?

Reply:
Yes, this is what the `one minute remaining` test checks as far as I can see.
But the name (especially taking the name of the second test `less then one minute remaining` into account) suggests (at least to me) that the test checks whether one minute input will lead to an output of one minute

Reply:
Ok, feel free to edit then 👍

Reply:
Pushed a commit reverting the display logic, but keeping the Unit test fix

Reply:
I looked into it and actually realised that there is a bug.
It is documented in #1504 and a fix is prepared in #1505